standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -1247,31 +1247,11 @@ public void createTable(Table tbl) throws AlreadyExistsException,
 
   public void createTable(Table tbl, EnvironmentContext envContext) throws AlreadyExistsException,
       InvalidObjectException, MetaException, NoSuchObjectException, TException {
-    if (!tbl.isSetCatName()) {
-      tbl.setCatName(getDefaultCatalog(conf));
```

Comment:
catalog name needs to be set if it is not set. Can you please add these lines back?

Reply:
Good catch ! 
I will add them back.